### PR TITLE
a

### PR DESCRIPTION
**Pull request by barosl2**

### CODE REVIEW

File: a.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+a
```
Comment (barosl2) [inline a.txt:1]: @barosl3 r+

**Comment by barosl3** [comment]
:hourglass: Testing commit 79a53cc with merge d38982d...

**Comment by barosl3** [comment]
:sunny: Test successful - [status](https://travis-ci.org/barosl2/test4/builds/93443371)

File: a.txt (hunk 1 above)
Comment (barosl2) [inline a.txt:1]: Hello!

Comment (barosl2) [inline a.txt:1]: World!

Comment (barosl2) [inline a.txt:1]: More!

Comment (barosl2) [inline a.txt:1]: Again!!!

Comment (barosl2) [inline a.txt:1]: Ralalala

Comment (barosl2) [inline a.txt:1]: ㅁㄴㅇㄹ ㅁㄴㅇㄹ

**Pull request merged**